### PR TITLE
Added Italian translation

### PR DESCRIPTION
**Pull request by alesssia**
Hi @suve, 
I've added the Italian translation, as per issue [#3](https://github.com/suve/vrms-rpm/issues/3).

### CODE REVIEW

File: man/it.man [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+e' stato accettato nel repository ufficiale di Fedora / CentOS - quali patenti
+software o restrizioni all'esportazione.
+.PP
+Richard Stallman e' stato, per anni, la figura piu' importante e piu' vocale
+nel movimento per il Free Software, ricordandoci incessantemente i principi 
+morali riguardanti il software. Come Richard, noi crediamo che gli utenti 
+debbano avere a disposizione un modo semplice per capire se stanno usando 
+software non libero nei loro sistemi. Questo programma e' un tentativo di 
+soddisfare questa esigenza.
+.SH OPTIONS
```
Comment (suve) [inline man/it.man]: The translation uses `[opzione]`. It's best to be consistent.

File: man/it.man [hunk 2]
```diff
@@ -0,0 +1,61 @@
+.TH VRMS-RPM 1
+.SH NAME
+vrms-rpm - Segnala i software non liberi installati 
+.SH SOMMARIO
+\fBvrms-rpm\fR \fI[OPTION]\fR ...
```
Comment (suve) [inline man/it.man]: `[OPTION]` should also be translated.

File: man/it.man [hunk 3]
```diff
@@ -0,0 +1,61 @@
+.TH VRMS-RPM 1
+.SH NAME
```
Comment (suve) [inline man/it.man]: This header looks like it's not translated.

File: lang/it.po [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+msgid "msg_total_free"
+msgstr "%d pacchetti liberi"
+msgid "msg_total_nonfree"
+msgstr "%d pacchetti non liberi"
+msgid "msg_rms_happy_ascii"
+msgstr "Solo pacchetti liberi - rms sarebbe orgoglioso!                           rms e' felice."
+msgid "msg_rms_happy"
+msgstr "Solo pacchetti liberi - rms sarebbe orgoglioso! "
+msgid "msg_rms_disappoint_ascii"
+msgstr "Piu' del 10%% dei pacchetti e' non libero. Odii la liberta'?              rms is deluso."
```
Comment (suve) [inline lang/it.po]: Please remove some spaces to make sure this line fits in 80 characters.

File: lang/it.po [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+msgid "msg_list_badargument"
+msgstr "%s: l'argomento per l'opzione --list deve essere uno tra: 'none', 'free', 'nonfree' o 'all'"
+msgid "msg_unknown_option"
+msgstr "%s: opzione sconosciuta '%s'"
+msgid "msg_total_free"
+msgstr "%d pacchetti liberi"
+msgid "msg_total_nonfree"
+msgstr "%d pacchetti non liberi"
+msgid "msg_rms_happy_ascii"
+msgstr "Solo pacchetti liberi - rms sarebbe orgoglioso!                           rms e' felice."
```
Comment (suve) [inline lang/it.po]: Please remove some spaces to fit this line in 79 characters.

Comment (alesssia) [reply lang/it.po]: Done!

File: lang/it.po (hunk 4 above)
Comment (alesssia) [reply lang/it.po]: The Italian translation is too long, I should remove part of the original sentence to be able to fit the Italian one in 80 characters :(

File: man/it.man (hunk 3 above)
Comment (alesssia) [reply man/it.man]: Sorry, I've missed it. Done.

File: man/it.man (hunk 2 above)
Comment (alesssia) [reply man/it.man]: Sorry, I've missed it. Done.

File: man/it.man (hunk 1 above)
Comment (alesssia) [reply man/it.man]: Sorry, I've missed it. Done.

File: lang/it.po (hunk 4 above)
Comment (suve) [reply lang/it.po]: It seems to me that we can just remove a couple spaces and it'll fit.

**Comment by suve** [comment]
Looks good to me. Merged. Thanks!

**Pull request merged**